djredcap/management/subcommands/fixture.py

```diff
@@ -9,6 +9,33 @@
 
 __project_name__ = ''
 
+# http://stackoverflow.com/a/6187936/407954

```

Comment:
Move this block of code to a separate module and remove the copy from `inspect.py`. I recommend creating a `djredcap/_csv.py`. The underscore is used so it does not conflict with the `csv` module in the standard library. Then import the needed reader in these two subcommands.